yt_dlp/extractor/indavideo.py

```diff
@@ -60,9 +74,6 @@ def _real_extract(self, url):
         elif isinstance(video_files, dict):
             video_urls.extend(video_files.values())
 
-        video_file = video.get('video_file')
-        if video:
-            video_urls.append(video_file)
```

Comment:
pretty sure it should've been `if video_file:`, but are we sure we're not losing any formats by removing this code?